std/jwt/signature.ts

```diff
@@ -0,0 +1,48 @@
+import type { Algorithm } from "./algorithm.ts";
+import { HmacSha256 } from "../hash/sha256.ts";
+import { HmacSha512 } from "../hash/sha512.ts";
+import { encode as convertUint8ArrayToBase64url } from "../encoding/base64url.ts";
+import { decodeString as convertHexToUint8Array } from "../encoding/hex.ts";
+
+export function convertHexToBase64url(input: string): string {
+  return convertUint8ArrayToBase64url(convertHexToUint8Array(input));
+}
+
+function encrypt(
+  alg: Algorithm | "none",
```

Comment:
`Algorithm` already includes `"none"`.

```suggestion
  alg: Algorithm,
```